WORKSPACE.bazel

```diff
@@ -10,3 +10,13 @@ http_archive(
 load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies")
 
 aspect_bazel_lib_dependencies()
+
+git_hash = "8fdf96b14cf4365ff5f8f2f4d5e418ff3e118715"
+archive_sha256 = "97e515e69fce3e59f2af35404179f9b893ea965d0e3576eb65ff5ec791834bbc"
+
+http_archive(
+    name = "bazel-orfs",
+    sha256 = archive_sha256,
+    strip_prefix = "bazel-orfs-%s" % git_hash,
+    url = "https://github.com/antmicro/bazel-orfs/archive/%s.tar.gz" % git_hash,
```

Comment:
How do I work with ORFS locally? I want to use ~/OpenROAD-flow-scripts/ to debug/hack ORFS. Add comment on how to do this, commented out example?

Reply:
This will work the same way as before. `bazel-orfs` still assumes that ORFS is under `~/OpenROAD-flow-scripts/`.

Reply:
OK. How do I work test with bazel-orfs locally?

Reply:
You can use `local_repoository` rule instead of `http_archive` and put there path to your local bazel-orfs repository.
Here is a documentation for that: https://bazel.build/reference/be/workspace#local_repository

Reply:
For now, we will only support the local workflow. Nobody will use the archive, so change to:

```
# git_hash = "8fdf96b14cf4365ff5f8f2f4d5e418ff3e118715"
# archive_sha256 = "97e515e69fce3e59f2af35404179f9b893ea965d0e3576eb65ff5ec791834bbc"
#
# http_archive(
#     name = "bazel-orfs",
#     sha256 = archive_sha256,
#     strip_prefix = "bazel-orfs-%s" % git_hash,
#     url = "https://github.com/The-OpenROAD-Project/bazel-orfs/archive/%s.tar.gz" % git_hash,
# )

# For now the repository is local, but we will switch to hosting bazel-orfs
# in https://github.com/The-OpenROAD-Project/ once it is ready.
local_repository(
    name = "bazel-orfs",
    path = "../bazel-orfs",
)
```

Reply:
Done